<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlPanelTextWidget</class>
 <widget class="QWidget" name="ControlPanelTextWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>258</width>
    <height>102</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QFontComboBox" name="fontComboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>160</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Шрифт</string>
       </property>
       <property name="maxVisibleItems">
        <number>10</number>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContents</enum>
       </property>
       <property name="minimumContentsLength">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="fontSizeBox">
       <property name="minimumSize">
        <size>
         <width>64</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Размер шрифта</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>150</number>
       </property>
       <property name="value">
        <number>14</number>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="posBtn">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Расположение</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QPushButton" name="textColorBtn">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="toolTip">
        <string>Цвет шрифта</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="boldBtn">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="toolTip">
        <string>Жирный</string>
       </property>
       <property name="text">
        <string>Ж</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="italicBtn">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="font">
        <font>
         <italic>true</italic>
        </font>
       </property>
       <property name="toolTip">
        <string>Курсив</string>
       </property>
       <property name="text">
        <string>К</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="underlineBtn">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="font">
        <font>
         <underline>true</underline>
        </font>
       </property>
       <property name="toolTip">
        <string>Подчёркивание</string>
       </property>
       <property name="text">
        <string>Ч</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="strikeBtn">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="font">
        <font>
         <strikeout>true</strikeout>
        </font>
       </property>
       <property name="toolTip">
        <string>Зачёркивание</string>
       </property>
       <property name="text">
        <string>abc</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QPushButton" name="symbolBtn">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Вставка символа</string>
       </property>
       <property name="text">
        <string>A$</string>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QPushButton" name="drawBtn">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Рисовать всегда</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <widget class="QPushButton" name="delBtn">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Удалить</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QGridLayout" name="gridLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="2">
      <widget class="QComboBox" name="ramkaBox">
       <property name="maximumSize">
        <size>
         <width>96</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Рамка</string>
       </property>
       <item>
        <property name="text">
         <string>Без рамки</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Рамка</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Заливка</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Рамка и Заливка</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="fonColorBtn">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="toolTip">
        <string>Цвет заливки</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QPushButton" name="okBtn">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Нанести</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="lockBtn">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="priorBox">
       <property name="maximumSize">
        <size>
         <width>64</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Приоритет</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>fontComboBox</tabstop>
  <tabstop>fontSizeBox</tabstop>
  <tabstop>boldBtn</tabstop>
  <tabstop>underlineBtn</tabstop>
  <tabstop>italicBtn</tabstop>
  <tabstop>strikeBtn</tabstop>
  <tabstop>drawBtn</tabstop>
  <tabstop>posBtn</tabstop>
  <tabstop>lockBtn</tabstop>
  <tabstop>ramkaBox</tabstop>
  <tabstop>delBtn</tabstop>
  <tabstop>okBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
